src/Common/Select.php

```diff
@@ -273,7 +271,7 @@ protected function addColWithAlias($spec)
      *
      * @param string $alias The column to remove
      *
-     * @return null
+     * @return boolean
```

Comment:
Inconsistency in return type : https://github.com/auraphp/Aura.SqlQuery/pull/110/files#diff-57cf575fdd7a21c19685563c9fa2a1c0L300